example/android/app/src/main/java/com/revenuecat/purchases_flutter_example/EmbeddingV1Activity.java

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases_flutter_example;
+
+import android.os.Bundle;
+import com.revenuecat.purchases_flutter.PurchasesFlutterPlugin;
+import io.flutter.app.FlutterActivity;
+import io.flutter.view.FlutterMain;
+
+public class EmbeddingV1Activity extends FlutterActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        FlutterMain.startInitialization(this);
+        super.onCreate(savedInstanceState);
+        PurchasesFlutterPlugin.registerWith(registrarFor("com.revenuecat.purchases_flutter"));
+    }
+}
```

Comment:
EmbeddingV1Activity.java uses the v1 embedding for the example project in the same folder as MainActivity to keep testing the v1 embedding’s compatibility with your plugin. Note that we have to manually register all the plugins instead of using GeneratedPluginRegistrant, as `MainActivity` used to do. Also copied from the upgrading docs.